_data/overlays/architecture.yml

```diff
@@ -31,7 +31,7 @@ CatONetworkDevices:
   description: Principles around maximum utilisation.
   link: '#NetworkDevices'
 
-CatOEmployeeDevices:
+CatOOnsiteEmployeeDevices:
```

Comment:
Probably want to have a section for Offsite devices that links to the same #EmployeeDevices ref.

Reply:
Is this true? I feel like if we're getting separate sections, they should cover different things, and it's not obvious to me that the architecture view should look at offsite devices. (Managing employee personal hardware is probably above this level in a business.)

Reply:
Just thinking in the context of the advice given in [architecture_strategy.md](https://github.com/ScottLogic/Technology-Carbon-Standard/blob/main/pages/views/roles/architecture_strategy.md), where it's focussed on choosing low-power devices/power management etc. where it will apply whether a company laptop is used in office or when working from home. Though maybe this is something that we want to look at specifically before we make these overlays public, as there are some decisions to be made around how you would display the two categories etc.

Reply:
I've added these to the overlays.